ui/src/util.ts

```diff
@@ -0,0 +1,19 @@
+export const rpcEndpoints = {
+  'agoric-local': {
+    osmosis: 'http://127.0.0.1:26655',
+    agoric: 'http://127.0.0.1:26657',
+  },
+  emerynet: {
+    osmosis: 'https://rpc.testnet.osmosis.zone:443',
+    agoric: 'https://emerynet.rpc.agoric.net',
+  },
+};
+
+export const ibcChannels = {
+  'agoric-local': {
+    osmosis: 'channel-1',
```

Comment:
> This would still have the same problem as the UI had before ...

Right; this is an issue that pre-dates this PR. Let's leave it out of scope by postponing it to a separate issue:

 - #68